Modules/socketmodule.c

```diff
@@ -2314,17 +2321,17 @@ cmsg_min_space(struct msghdr *msg, struct cmsghdr *cmsgh, size_t space)
     #endif
     if (msg->msg_controllen < 0)
         return 0;
+    if (space < cmsg_len_end)
```

Comment:
This is mistake will have recall the pull request and move this back where it was orignally